### ISSUE TITLE
Bypass Temporary Stop signal

### ISSUE THREAD

**Issue by PiggyWhiskey** [Yousei9/Logistic-Train-Network#318]
**TL;DR**
Allow LTN Stops to accept a signal which prevents Requester/Provider from generating temporary stops

**What ?**
Allow LTN Stops to accept a signal which prevents Requester/Provider from generating temporary stops
For provider or requesters with this signal, the temporary stop preceding it should not be generated.

**Why ?**
This would be mostly valuable on Requester stops, allowing numerous resource collection stops to share stations, as well as using LTN functionality to share locomotives.

For example, 10 Trains, 5 Requester (With signal), and 2 Iron Provider, 2 Copper Provider, 2 Coal Providers.
This will allow all 5 requester stops to unload Iron/Copper/Coal and the player can place a stacker before entry for a far greater throughput.
Current limitations require sharing requests and inventory between multiple stops.

For a Provider example,
2 Stations parallel to each other can provide Iron, and the next train coming in will route to the next free Provider, rather than wait behind the in-use provider.

This is similar to the Waypoint feature requested https://github.com/Yousei9/Logistic-Train-Network/issues/287
But should be easier to implement (If requester/provider has signal, do not generate temporary stop)
And will enhance LTN to allow networks to scale up to larger values needed in City Grid playstyles.

**Comment by NightOps007** [comment]
If you're planning to stack trains, then you might as well use the vanilla train logic.

**Comment by Yousei9** [comment]
Use vanilla logic if you want vanilla logic to decide which identically named station to use.

LTN operates per stop and uses those temp stops to ensure trains will always path to the intended stop.
